NAMESPACE

```diff
@@ -1,12 +1,32 @@
-import(AnnotationHub)
-import(AnnotationHubData)
-import(Biostrings)
-import(BiocManager)
+# Generated by roxygen2: do not edit by hand
 
-importFrom("Biobase", "addVigs2WinMenu")
-importFrom("BiocManager", "version")
-importFrom("GenomicRanges", "GRanges")
-importFrom("utils", "packageVersion", "packageDescription", "vignette", 
-           "download.file")
-
-export(EuPathDB, availableEuPathDB)
```

Comment:
It looks like your changes are clobbering the AH-related NAMESPACE entries. Some of the functions are used in R/EuPathDB.R, so these could probably be moved there so that they are automatically picked up.. Not sure about the rest of the functions (which are used in scripts); these may need to be manually added. I imagine we could just specify them in some location that roxygen will know to pick them up when it is generating the NAMESPACE file.

Reply:
That is fine by me.  I am thinking that once all the pieces are in place, the code should  be reorganized so that either more or less material is located in EuPathDB.R.  Once that global question is decided, I think it will define how to handle the NAMESPACE/roxygen.